wrappers/python/indy/agent.py

```diff
@@ -0,0 +1,308 @@
+import asyncio
+import logging
+from ctypes import *
+from typing import List, Tuple, Any
+
+from .error import ErrorCode, IndyError
+from .libindy import do_call, create_cb
+
+
+class Event:
+    handle: int
+    error: IndexError
+
+    def is_success(self):
+        return self.error is None
+
+
+class ConnectionEvent(Event):
```

Comment:
Add docstring with descriptions of these classes, public methods, functions and their params